src/lib/y2storage/yaml_writer.rb

```diff
@@ -72,13 +72,20 @@ def write(devicegraph, yaml_file)
     # @return [Array<Hash>]
     def yaml_device_tree(devicegraph)
       yaml = []
-      devicegraph.disk_devices.each { |device| yaml << yaml_disk_device(device) }
+      devices(devicegraph).each { |device| yaml << yaml_disk_device(device) }
       devicegraph.lvm_vgs.each { |lvm_vg| yaml << yaml_lvm_vg(lvm_vg) }
       yaml
     end
 
   private
 
+    # Devices that will be converted to yaml format
+    #
+    # @return [Array<Y2Storage::Device>]
```

Comment:
Maybe we need a more descriptive name here. Something like `root_devices` or `top_level_devices`...